maliput_cargo/src/road_geometry.rs

```diff
@@ -0,0 +1,90 @@
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+// DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+// This example demonstrates how to create a RoadNetwork object from an OpenDRIVE file and
+// exercise the RoadGeometry API.
+fn main() {
+    use std::collections::HashMap;
+
+    // Get location of odr resources
+    let package_location = std::env::var("CARGO_MANIFEST_DIR").unwrap();
+    let xodr_path = format!("{}/resources/TShapeRoad.xodr", package_location);
+
+    let road_network_properties = HashMap::from([
+        ("road_geometry_id", "my_rg_from_rust"),
+        ("opendrive_file", xodr_path.as_str()),
+        ("linear_tolerance", "0.001"),
+    ]);
+
+    let road_network =
+        maliput::api::RoadNetwork::new("maliput_malidrive", &road_network_properties);
+    let road_geometry = road_network.road_geometry();
+
+    // Exercise the RoadGeometry API.
+    println!("linear_tolerance: {}", road_geometry.linear_tolerance());
+    println!("angular_tolerance: {}", road_geometry.angular_tolerance());
+    println!("num_junctions: {}", road_geometry.num_junctions());
+
+    let lanes = road_geometry.get_lanes();
+    println!("num_lanes: {}", lanes.len());
+    println!("lanes: ");
+    for lane in lanes {
+        println!("\tlane id: {}", lane.id());
+    }
+
+    // Call to_road_position() method to convert an INERTIAL position to a LANE position.
+    // This inertial position is outside the road volume.
+    let inertial_position = maliput::api::InertialPosition::new(-0.5, 0., 1.0);
+    let road_position_result = road_geometry.to_road_position(&inertial_position);
+
+    let associated = road_position_result.distance < road_geometry.linear_tolerance();
+    println!("");
+    println!("Candidate Point: {:?}", inertial_position);
+    println!("\tInside Road Volume?: {}", associated);
+    if !associated {
+        println!(
+            "\tNearest Point: {:?}",
+            road_position_result.nearest_position
+        );
+        println!(
+            "\tNearest Point - Distance: {}",
+            road_position_result.distance
+        );
+    }
+    println!(
+        "\tLane ID: {}",
+        road_position_result.road_position.lane().id()
+    );
+    println!(
+        "\tLane Position: {}",
+        road_position_result.road_position.pos()
+    );
+    println!("");
+}
```

Comment:
Shouldn't we return `0` here?
```suggestion
    println!("");
    
    return 0;
}
```

Reply:
There is no need to return anything in the main function however we could return if we want a `Result<>` in that case we would do something like: 
```rust
fn main() -> Result<(), Box<dyn std::error::Error>> {
    if bla bla {
      Error{"}  
    }
    Ok(())

}
```
See https://doc.rust-lang.org/rust-by-example/error/result.html#using-result-in-main

However in this case we are not checking an error condition so I would leave it this way for the moment